docs/source/quickstart.rst

```diff
@@ -0,0 +1,108 @@
+Quickstart
```

Comment:
@madsbk what do you think about this example for a quickstart ?

Reply:
Looks good; I have added some suggestions.